src/test/java/com/linagora/dav/contracts/CalJsonContract.java

```diff
@@ -281,197 +287,4368 @@ void reportShouldListEvents() {
             .request(HttpMethod.valueOf("REPORT"))
             .uri("/calendars/" + testUser.id() + "/" + testUser.id() + ".json")
             .send(body("""
-                {"match":{"start":"20250201T000000","end":"20250215T000000"}}""")));
+                {"match":{"start":"20250201T000000","end":"20250215T000000"}}""")));
+
+        assertThatJson(response.body())
+            .whenIgnoringPaths("_embedded.dav:item[0].data[1][0][3]")  // ignore prodid
+            .isEqualTo(String.format("""
+                {
+                    "_links": {
+                        "self": {
+                            "href": "/calendars/%s/%s.json"
+                        }
+                    },
+                    "_embedded": {
+                        "dav:item": [
+                            {
+                                "_links": {
+                                    "self": {
+                                        "href": "/calendars/%s/%s/abcd.ics"
+                                    }
+                                },
+                                "etag": "\\"8c97fb06c60212c47d46a9c8c0f625ef\\"",
+                                "data": [
+                                    "vcalendar",
+                                    [
+                                        [
+                                            "prodid",
+                                            {},
+                                            "text",
+                                            "-//Sabre//Sabre VObject 4.1.3//EN"
```

Comment:
```suggestion
                                            "${json-unit.ignore}"
```

if you like annotation